LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationConfigurationService.cs

```diff
@@ -34,6 +35,12 @@ internal sealed class BasicsStationConfigurationService : IBasicsStationConfigur
 
         public void Dispose() => this.cacheSemaphore.Dispose();
 
+        public async Task<Region> GetRegionFromBasicsStationConfiguration(StationEui stationEui, CancellationToken cancellationToken)
+        {
+            var config = await this.GetRouterConfigMessageAsync(stationEui, cancellationToken);
+            return LnsStationConfiguration.GetRegion(config);
```

Comment:
How often do we expect that this is called, and at which point in time? The cache of the configuration currently lives for two hours - if we expect to deviate significantly from that we might want to think about changing the caching strategy for the underlying config/for the region itself.

Reply:
As discussed, let's put this topic on tomorrow's parking lot.